src/support_sphere_py/src/support_sphere/scripts/execute_sql_statement.py

```diff
@@ -101,8 +107,16 @@
 COMMIT;
 """
 
+
 # Execute the SQL commands
-def execute_custom_sql_statement(sql_text):
+@execute_sql_app.command(help="CLI command to execute custom SQL statement passed as `sql_text` argument in multi-line string")
```

Comment:
I think this function doesn't need to be exposed. Most likely if user wants to execute a sql statement, they would point to a sql file input rather than a straigt up multi-line string as this can be too much to paste in the terminal.

Reply:
Updated.